functions/AutoRemediateEC2-006.js

```diff
@@ -21,11 +20,11 @@ module.exports.handler = (event, context, callback) => {
       return handleError(err.message ? err.message : 'removing postgresql database access on port 5432 failed')
     }
     console.log('Result', result)
-    return callback(null, 'Successfully processed event')
+    return 'Successfully processed event'
```

Comment:
🔧 This needs to use `callback` because this is inside of a `callback` on the `AccessRevoker` method.

Same for the`handleError` method below